app/views/partials/dependency_section.scala.html

```diff
@@ -65,8 +66,8 @@
         </span>
         <span id="@{dependency.name}-@{rootId}-bobbyrule" class="col-xs-2">
         @dependency.versionState match {
-            case Some(VersionState.BobbyRulePending (violation)) => { <a id="see-pending-bobby-rule-@dependency.group:@dependency.name" href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
-            case Some(VersionState.BobbyRuleViolated(violation)) => { <a href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
+            case Some(VersionState.BobbyRulePending (violation)) => { <a id="see-pending-bobby-rule-@dependency.group:@dependency.name" href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{UrlUtils.encodeQueryParam(violation.range.range)}">See rule</a> }
```

Comment:
jfyi
```
Bad value “/bobbyrules#uk.gov.hmrc:reactivemongo-test:[4.0.0,4.4.0]” for attribute “href” on element “a”: Illegal character in fragment: “[” is not allowed.
```

Reply:
presumably these ids were added for auditing, and have no impact on acceptance tests

Reply:
yeah, acceptance tests are passing :)